mod/01_rating/set/type/metric_value.rb

```diff
@@ -79,17 +84,23 @@ def create_source
 
 format :html do
   def default_new_args args
-    args[:hidden] = {
-      :success=>{:id=>'_self', :soft_redirect=>true, :view=>:titled},
-      'card[subcards][+metric][content]' => args[:metric]
-    }
-
+    if !args[:source]
+      args[:hidden] = {
+        :success=>{:id=>'_self', :soft_redirect=>true, :view=>:titled},
+        'card[subcards][+metric][content]' => args[:metric]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.